src/java/us/kbase/templates/python_server.vm.properties

```diff
@@ -146,12 +147,6 @@ class JSONRPCServiceCustom(JSONRPCService):
         debugging purposes.
         """
         rdata = jsondata
-        # we already deserialize the json string earlier in the server code, no
-        # need to do it again
-#        try:
-#            rdata = json.loads(jsondata)
-#        except ValueError:
-#            raise ParseError
```

Comment:
A bit of background here - this method is a copy of `jsonrpcbase.call_py` that overrides the original method with a few changes around error handling. I left the commented out code in place so that a later programmer could easily see what changes were made from the parent method and why those changes were made.